server/data/placementRequestClient.ts

```diff
@@ -24,11 +29,25 @@ export default class PlacementRequestClient {
     >
   }
 
-  async dashboard(isParole: boolean): Promise<Array<PlacementRequest>> {
-    return (await this.restClient.get({
+  async dashboard(
+    isParole: boolean,
+    page: number = 1,
```

Comment:
v minor but ESLint is complaining that there's no need to define a type here as it's inferred
```suggestion
    page = 1,
```